src/components/ha-sidebar.ts

```diff
@@ -647,6 +666,11 @@ class HaSidebar extends LitElement {
         padding: 4px;
         font-weight: 500;
       }
+
+      .flipVertical {
```

Comment:
Why does this needs to be a class name instead of selecting based on `:host([rtl])` ?